src/shared/components/window/WindowStore.ts

```diff
@@ -20,13 +21,14 @@ class WindowStore {
     @action
     private setWindowSize() {
         this.size = {
-            width: this.windowObj.innerWidth,
-            height: this.windowObj.innerHeight
+            width: Math.max(this.windowObj.innerWidth, this.documentObj.documentElement.scrollWidth),
```

Comment:
I'm curious why we do this.  Let me know when you get in.  It does appear to be working well!